cmake/projects/aglet/hunter.cmake

```diff
@@ -53,6 +53,17 @@ hunter_add_version(
     bbd0c7ec9e5b559aba05aabdeffefb843f76edb8
 )
 
+hunter_add_version(
+    PACKAGE_NAME
+    aglet
+    VERSION
+    1.2.2
+    URL
+    "https://github.com/elucideye/aglet/archive/v1.2.2.tar.gz"
```

Comment:
This version is not in Hunter master branch yet. If new version will broke some old toolchains then toolchains should be removed for old version first:
* https://docs.hunter.sh/en/latest/creating-new/update.html#remove-toolchains